tests/test_integration.py

```diff
@@ -22,6 +22,15 @@ def test_match_spectra_from_path(mgf_path, spec2vec_client):
     assert len(result) == 377
 
 
+@pytest.mark.internet_connection
+@pytest.mark.skip(reason="Requires a valid token")
+def test_match_spectra_from_path_negative_mode(mgf_path, spec2vec_client):
+    result = spec2vec_client.match_spectra_from_path(mgf_path, 10, "negative")
+
+    assert result
+    assert len(result) == 377
```

Comment:
how do we know this number in advance?? this asset will never change?

Reply:
yes the asset in question has 377 spectra so we should get one result per spectra